BonusCalcListener.Tests/UseCase/PayElementMapperTests.cs

```diff
@@ -223,12 +263,47 @@ public void ShouldCreateOvertimePayElementWithZeroValue()
             evtData.IsOutOfHours = false;
             evtData.IsOvertime = true;
             evtData.OperativeCost = 0;
+            evtData.PaymentType = null;
+
+            // Act
+            var result = _sut.BuildPayElement(evtData);
+
+            // Assert
+            result.PayElementTypeId.Should().Be(PayElementTypeIds.Overtime);
+            result.WorkOrder.Should().Be(evtData.WorkOrderId);
+            result.Address.Should().Be(evtData.Address);
+            result.Comment.Should().Be(evtData.Description);
+            result.ClosedAt.Should().Be(evtData.ClosedTime);
+            result.Monday.Should().Be(0.0m);
+            result.Tuesday.Should().Be(0.0m);
+            result.Wednesday.Should().Be(0.0m);
+            result.Thursday.Should().Be(0.0m);
+            result.Friday.Should().Be(0.0m);
+            result.Saturday.Should().Be(0.0m);
+            result.Sunday.Should().Be(0.0m);
+            result.Duration.Should().Be(0.0m);
+            result.Value.Should().Be(0.0m);
+        }
+
+        [Test]
+        public void WhenPaymentTypeIsOvertime_ShouldCreateOvertimePayElementWithZeroValue()
+        {
+            // Arrange
+            var evtData = _fixture.Create<WorkOrderOperativeSmvData>();
+            evtData.WorkOrderStatusCode = 1000;
+            evtData.StandardMinuteValue = 0;
+            evtData.JobPercentage = 50;
+            evtData.ClosedTime = new DateTime(2021, 11, 08, 14, 32, 01);
+            evtData.IsOutOfHours = false;
+            //evtData.IsOvertime = true;
```

Comment:
Should this be explicitly set to false as that's the column default in Repairs Hub?